cmd/kubeadm/app/util/apiclient/idempotency.go

```diff
@@ -283,8 +283,10 @@ func PatchNodeOnce(client clientset.Interface, nodeName string, patchFn func(*v1
 		// First get the node object
 		n, err := client.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
 		if err != nil {
-			// TODO this should only be for timeouts
-			return false, nil
+			if apierrors.IsTimeout(err) {
+				return false, nil // only retry when timeout.
+			}
+			return false, err
```

Comment:
the current error retries on failed Get on Nodes.
it is actually safer to retry on *any* error, so i think i disagree with the original comment.

we might as well remove it.

Reply:
Done